extensions/resource-deployment/src/ui/deployClusterWizard/pages/serviceSettingsPage.ts

```diff
@@ -400,6 +401,7 @@ export class ServiceSettingsPage extends WizardPageBase<DeployClusterWizard> {
 
 	public onLeave(): void {
 		setModelValues(this.inputComponents, this.wizard.model);
+		Object.assign(this.wizard.inputComponents, this.inputComponents);
```

Comment:
what is this line for?

Reply:
it copies the inputComponents of the page to the inputComponents of the whole wizard.